data/categories.txt

```diff
@@ -1724,7 +1735,9 @@ sw:Duka la nguo|nguo
 fa:لباس فروشی|بوتیک
 
 shop-car
-en:Car Shop|U+1F697|U+1F698|U+1F699
+en:Auto dealer|U+1F697|U+1F698|U+1F699
+en-GB:Car dealer|Car showroom|U+1F697|U+1F698|U+1F699
+en-AU:Car dealer|Car showroom|U+1F697|U+1F698|U+1F699
```

Comment:
Capitalized Dealer in all 3 cases will look better on the screen.

Reply:
Sure, this is a matter of style. I disagree that it looks better, and in all other languages it isn't capitalised in this way, but I'll make this change.